### ISSUE TITLE
Nos. of images in the Val Set

### ISSUE THREAD

**Issue by Viresh-R** [lvis-dataset/lvis-api#32]
## ❓ Questions and Help
Hey folks,
Looks like there are 5K images in the val2017 set available here: http://images.cocodataset.org/zips/val2017.zip
The v1.0 description, however, says that there are 19,809 images in the val set.

Not sure if I am using any outdated link for the val set ?

Regards,
Viresh

**Comment by atiorh** [comment]
@agrimgupta92 could you please address this?

**Comment by atiorh** [comment]
The validation set images download link on https://www.lvisdataset.org/dataset points to 5k images but the link text reads `[19,809 images](http://images.cocodataset.org/zips/val2017.zip) (1 GB)`.  This is clearly inconsistent with either of the test sets' link text: `[19,822 images](http://images.cocodataset.org/zips/test2017.zip) (6 GB)` unless the per image size is 6x different..

**Comment by rbgirshick** [comment]
Please see the note on that page, reproduced below:

> Note: LVIS uses the COCO 2017 train, validation, and test image sets. If you have already downloaded the COCO images, you only need to download the LVIS annotations. LVIS val set contains images from COCO 2017 train in addition to the COCO 2017 val split.

The last sentence describes how the (19,809) LVIS val images come from a combination of the COCO 2017 train and val splits.

**Comment by Viresh-R** [comment]
thanks for the clarification @rbgirshick !
